src/nikonmn_int.cpp

```diff
@@ -1662,7 +1645,7 @@ namespace Exiv2 {
                 // selected point was not the actually used one
                 // (Roger Larsson: my interpretation, verify)
                 os << sep;
-                for (unsigned fpid=0; fpid<focuspoints; fpid++)
+                for (unsigned fpid = 0; fpid < nikonFocuspoints.size(); fpid++)
```

Comment:
[suggestion] use `size_t` or `auto` instead of `unsigned`

Reply:
Done.